autoroute/main.go

```diff
@@ -47,13 +56,16 @@ func GetLinkLocalAddr(dev net.Interface) (*net.IPAddr, error) {
 	}
 
 	if ll_addr == nil {
-		log.Fatalf("Couldn't find interface link-local addresses %v", dev.Name)
+		msg := fmt.Sprintf("Couldn't find link-local address on interface %v", dev.Name)
+		e := LinkLocalError{msg, false}
```

Comment:
I guess it should be fatal if *none* of the interfaces have link locals. But if I only have a wireless connection of course I'm not going to get a link local on my ethernet device.